fake_filesystem_unittest_test.py

```diff
@@ -141,6 +141,22 @@ def test_fakepathlib(self):
         else:
             self.assertTrue(self.fs.Exists('/fake_file.txt'))
 
+    def test_disk_usage_on_file_creation(self):
+        total_size = 100
+        self.fs.AddMountPoint('mount', total_size)
+        def create_too_large_file():
+            with open('/mount/file', 'w') as dest:
+                dest.write('a' * (total_size + 1))
+
+        self.assertRaises((OSError, IOError), create_too_large_file)
+
+        self.assertEqual(0, self.fs.GetDiskUsage('/mount').used)
+
+        with open('/mount/file', 'w') as dest:
+            dest.write('a' * total_size)
+
+        self.assertEqual(total_size, self.fs.GetDiskUsage('/mount').used)
+
```

Comment:
Looks nice - thanks! 
Could you please move this test to the class _DiskSpaceTest_ in _fake_filesystem_test.py_ (_fake_filesystem_unitest_test.py_ is used for unit test/patcher related tests) and rename it to conform to the used style (e.g. camel case for test names - seems to be Google's coding style) - I will merge it afterwards.